src/components/styled-components/Typography.js

```diff
@@ -49,24 +49,31 @@ export default function Typography({ children, variant, component, ...rest }) {
 
 const StyledH1 = styled.h1`
 	font-size: ${({ size }) => size};
+	font-family: Questrial;
 `;
```

Comment:
When every typography style you created uses this font-family, you could set it within your globalstyle for the whole app.